omod-2.2/pom.xml

```diff
@@ -0,0 +1,186 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.openmrs.module</groupId>
+		<artifactId>webservices.rest</artifactId>
+		<version>2.22.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>webservices.rest-omod-2.2</artifactId>
+	<packaging>jar</packaging>
+	<name>Rest Web Services 2.2 OMOD</name>
+	<description>OpenMRS module project for Rest Web Services</description>
+
+	<properties>
+		<openmrs.version.2.1>2.1.0</openmrs.version.2.1>
```

Comment:
The openmrs version should be 2.2.0-SNAPSHOT because that is where your program enrollment changes are.

Reply:
Cool, let me change that.